shared/actions/chat2/index.js

```diff
@@ -2920,6 +2928,15 @@ function* chat2Saga(): Saga.SagaGenerator<any, any> {
     EngineGen.chat1NotifyChatNewChatActivity,
     onNewChatActivity
   )
+  yield* Saga.chainAction<EngineGen.Chat1ChatUiChatShowManageChannelsPayload>(
+    EngineGen.chat1ChatUiChatShowManageChannels,
+    onChatShowManageChannels
+  )
+
+  yield* Saga.chainAction<ConfigGen.SetupEngineListenersPayload>(
```

Comment:
why?

Reply:
Makes the delegated chat UI work in the service.